src/DependencyInjection/ContentApiConfiguration.php

```diff
@@ -49,6 +49,10 @@ private function getServicesDefinition() : ArrayNodeDefinition
                         ->isRequired()
                         ->info('The service name to use.')
                     ->end()
+                    ->booleanNode('include_prefix')
+                        ->defaultTrue()
+                        ->info('Whether to include the service prefix in the routes.')
```

Comment:
Should be possible to add validation that if this is `false` there's only one service.